examples/two_mass_spring_damper/run.py

```diff
@@ -0,0 +1,94 @@
+import matplotlib.pyplot as plt
+from numpy import linspace
+from pydy.system import System
+from sympy import symbols, Matrix
+from sympy.physics.mechanics import dynamicsymbols, eombase
+
+# Define the dynamic symbols
+
+x1, x2, u1, u2 = dynamicsymbols('x1 x2 u1 u2')
+
+# Define the constant symbols
+
+m1, c1, k1 = symbols('m1 c1 k1')
+m2, c2, k2 = symbols('m2 c2 k2')
+
+# Define the mass matrix and forcing vector
+
+mm = Matrix([[m1,  0],
+             [0,  m2]])
+f = Matrix([-(k1+k2)*x1 + k2*x2 - (c1+c2)*u1 + c2*u2,
+            k2*x1 - k2*x2 + c2*u1 - c2*u2])
+
+mm_full = Matrix([[1, 0,  0,  0],
+                  [0, 1,  0,  0],
+                  [0, 0, m1,  0],
+                  [0, 0,  0, m2]])
+f_full = Matrix([u1,
+                 u2,
+                 -(k1+k2)*x1 + k2*x2 - (c1+c2)*u1 + c2*u2,
+                 k2*x1 - k2*x2 + c2*u1 - c2*u2])
+
+# Form the rhs of q' = G(q, u, t, r, p). Kinematic equation
+
+G = Matrix([u1, u2])
+
+# Form the rhs of the dynamics equations
+
+RHS = mm_full.inv()*f_full
+
+# Form the remaining input lists for the equations of motion class
+
+coordinates = (x1, x2)
+speeds = (u1, u2)
+states = (x1, x2, u1, u2)
+
+# Form the kinetic energy to attach to one of the EOM instances as an output
+# equation
+
+KE = 0.5 * (m1*u1**2 + m2*u2**2)
+out_eqns = {"kinetic_energy": KE}
```

Comment:
`Most things are passed into the *Methods classes as:... expri = 0`

I think that the form `out_eqns = {'Kinetic Energy': [expr1, expr2, ...]}` is a good combination of the previous API where `expri = 0` still holds and you can differentiate between the basic equations you put in. I think `out_eqns= [expr1, expr2, expr3, ...]` would be too vague because there is a variety of different output equations you could want to specify (for example expr1 = kinetic energy of body 2, expr2 = contact forces in joint3, expr4 = auxiliary equation).

`In the System class we use mostly dictionaries ... Where you can now include other symbols that represent the output equations.`

Might it be that strings or symbols could be used for the keys in the dictionary? I think strings might be simpler because some of the output equations could be specified by the `*Methods` class in the `*Methods.to_eom()` method (like Kane's auxiliary equations) and the user would have to come up with the symbol that the method used instead of just a string with the label for the expression (which brings up the need to be able to pass additional equations to `output_eqns` outside of `EOM.__init__(...)`).